app/src/androidTest/java/org/mozilla/fenix/ui/SearchTest.kt

```diff
@@ -37,14 +38,14 @@ class SearchTest {
         }
     }
 
+    @Ignore("This test cannot run on virtual devices due to camera permissions being required")
```

Comment:
@npark-mozilla, it looks like the devices on Firebase have limited camera permissions.  We can probably investigate this further later but, for now, it seems reasonable to me that we commit these tests and Ignore so they can be run locally as needed.

Reply:
yes that's fine, since we should only run tests on physical device on firebase.